<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>591</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>170</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>170</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>170</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>FreeSans</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>SDMed2</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabPrincipal">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <widget class="QMenu" name="menuImportar">
     <property name="title">
      <string>Importar</string>
     </property>
     <addaction name="actionImportarFIEBCD"/>
    </widget>
    <widget class="QMenu" name="menuExportar">
     <property name="title">
      <string>Exportar</string>
     </property>
     <addaction name="actionExportarFIEBCD"/>
    </widget>
    <widget class="QMenu" name="menuAbrir_reciente">
     <property name="title">
      <string>Abrir reciente</string>
     </property>
    </widget>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrirBBDD"/>
    <addaction name="menuAbrir_reciente"/>
    <addaction name="actionCerrar"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_como"/>
    <addaction name="separator"/>
    <addaction name="menuImportar"/>
    <addaction name="menuExportar"/>
    <addaction name="separator"/>
    <addaction name="actionPropiedades_de_la_obra"/>
    <addaction name="separator"/>
    <addaction name="actionObra_1"/>
    <addaction name="actionObra_2"/>
    <addaction name="actionObra_3"/>
    <addaction name="actionObra_4"/>
    <addaction name="actionObra_5"/>
    <addaction name="action_Salir"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdicion">
    <property name="title">
     <string>&amp;Edicion</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionDeshacer"/>
    <addaction name="actionRehacer"/>
    <addaction name="separator"/>
    <addaction name="actionCopiar"/>
    <addaction name="actionCortar"/>
    <addaction name="actionPegar"/>
    <addaction name="actionMover"/>
    <addaction name="separator"/>
    <addaction name="actionSeleccionar_todo"/>
    <addaction name="actionAnular_selecci_n"/>
    <addaction name="separator"/>
    <addaction name="actionBuscar"/>
    <addaction name="actionReemplazar"/>
    <addaction name="separator"/>
    <addaction name="actionAnterior"/>
    <addaction name="actionSiguiente"/>
    <addaction name="separator"/>
    <addaction name="actionSubir_nivel"/>
    <addaction name="actionBajar_nivel"/>
    <addaction name="separator"/>
    <addaction name="actionAdelante"/>
    <addaction name="actionAtras"/>
    <addaction name="separator"/>
    <addaction name="actionAvanzar_linea"/>
    <addaction name="actionRetroceder_l_inea"/>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>&amp;Ver</string>
    </property>
    <addaction name="actionConceptos"/>
   </widget>
   <widget class="QMenu" name="menuHerraminetas">
    <property name="title">
     <string>&amp;Herramientas</string>
    </property>
    <addaction name="actionAjustar_precio"/>
   </widget>
   <widget class="QMenu" name="menuInformes">
    <property name="title">
     <string>&amp;Informes</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAcerca_de"/>
    <addaction name="actionAcerca_de_Qt"/>
   </widget>
   <widget class="QMenu" name="menuVentana">
    <property name="title">
     <string>Ventana</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEdicion"/>
   <addaction name="menuVer"/>
   <addaction name="menuHerraminetas"/>
   <addaction name="menuVentana"/>
   <addaction name="menuInformes"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="BarraPrincipal">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevo"/>
   <addaction name="actionAbrirBBDD"/>
   <addaction name="actionImportar"/>
   <addaction name="actionExportar"/>
   <addaction name="actionGuardar"/>
   <addaction name="actionCerrar"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="EditlBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDeshacer"/>
   <addaction name="actionRehacer"/>
   <addaction name="actionCopiar"/>
   <addaction name="actionCortar"/>
   <addaction name="actionPegar"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="MovBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionAtras"/>
   <addaction name="actionIr_a_inicio"/>
   <addaction name="actionAdelante"/>
   <addaction name="actionVer_Arbol"/>
  </widget>
  <widget class="QToolBar" name="CertBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Nuevo...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAbrirBBDD">
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/database.png</normaloff>:/images/database.png</iconset>
   </property>
   <property name="text">
    <string>Abrir...</string>
   </property>
   <property name="toolTip">
    <string>Abrir de la BBDD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/floppy.png</normaloff>:/images/floppy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionGuardar_como">
   <property name="text">
    <string>Guardar como...</string>
   </property>
  </action>
  <action name="actionPropiedades_de_la_obra">
   <property name="text">
    <string>Propiedades de la obra...</string>
   </property>
  </action>
  <action name="action_Salir">
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionCortar">
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cortar</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionPegar">
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Pegar</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionMover">
   <property name="text">
    <string>Mover...</string>
   </property>
  </action>
  <action name="actionSeleccionar_todo">
   <property name="text">
    <string>Seleccionar todo</string>
   </property>
  </action>
  <action name="actionAnular_selecci_n">
   <property name="text">
    <string>Anular selección</string>
   </property>
  </action>
  <action name="actionBuscar">
   <property name="text">
    <string>Buscar...</string>
   </property>
  </action>
  <action name="actionReemplazar">
   <property name="text">
    <string>Reemplazar</string>
   </property>
  </action>
  <action name="actionAnterior">
   <property name="text">
    <string>Anterior</string>
   </property>
  </action>
  <action name="actionSiguiente">
   <property name="text">
    <string>Siguiente</string>
   </property>
  </action>
  <action name="actionSubir_nivel">
   <property name="text">
    <string>Subir nivel</string>
   </property>
  </action>
  <action name="actionBajar_nivel">
   <property name="text">
    <string>Bajar nivel</string>
   </property>
  </action>
  <action name="actionAdelante">
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/Adelante.png</normaloff>:/images/Adelante.png</iconset>
   </property>
   <property name="text">
    <string>Adelante</string>
   </property>
  </action>
  <action name="actionAtras">
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/Atras.png</normaloff>:/images/Atras.png</iconset>
   </property>
   <property name="text">
    <string>Atras</string>
   </property>
  </action>
  <action name="actionAvanzar_linea">
   <property name="text">
    <string>Avanzar línea</string>
   </property>
  </action>
  <action name="actionRetroceder_l_inea">
   <property name="text">
    <string>Retroceder línea</string>
   </property>
  </action>
  <action name="actionConceptos">
   <property name="text">
    <string>Conceptos</string>
   </property>
  </action>
  <action name="actionAjustar_precio">
   <property name="text">
    <string>Ajustar...</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de SDMed</string>
   </property>
  </action>
  <action name="actionAcerca_de_Qt">
   <property name="text">
    <string>Acerca de Qt</string>
   </property>
  </action>
  <action name="actionImportarFIEBCD">
   <property name="text">
    <string>FIEBCD</string>
   </property>
  </action>
  <action name="actionExportarFIEBCD">
   <property name="text">
    <string>FIEBCD</string>
   </property>
  </action>
  <action name="actionRecientes">
   <property name="text">
    <string>Recientes</string>
   </property>
  </action>
  <action name="actionObra_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Obra 1</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionObra_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Obra 2</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionObra_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Obra 3</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionObra_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Obra 4</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionObra_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Obra 5</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDeshacer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/deshacer.png</normaloff>:/images/deshacer.png</iconset>
   </property>
   <property name="text">
    <string>Deshacer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionRehacer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/rehacer.png</normaloff>:/images/rehacer.png</iconset>
   </property>
   <property name="text">
    <string>Rehacer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionIr_a_inicio">
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/SubirTodo.png</normaloff>:/images/SubirTodo.png</iconset>
   </property>
   <property name="text">
    <string>Ir a inicio</string>
   </property>
   <property name="toolTip">
    <string>Ir a inicio</string>
   </property>
  </action>
  <action name="actionVer_Arbol">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/tree.png</normaloff>:/images/tree.png</iconset>
   </property>
   <property name="text">
    <string>Ver Arbol</string>
   </property>
  </action>
  <action name="actionImportar">
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/importar.png</normaloff>:/images/importar.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Importar</string>
   </property>
   <property name="toolTip">
    <string>Importar</string>
   </property>
  </action>
  <action name="actionExportar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/images/exportar.png</normaloff>:/images/exportar.png</iconset>
   </property>
   <property name="text">
    <string>Exportar</string>
   </property>
   <property name="toolTip">
    <string>ExportarBC3</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
